framework/key-store/dynamodb-key-storage-interface.md

```diff
@@ -19,7 +19,7 @@
 
 ## Overview
 
-The Dynamodb Encrypted Key Store is the default implementation of the [encrypted key store interface](./encrypted-key-store.md#overview)
+The Dynamodb Key Storage Interface is the default implementation of the [encrypted key store interface](./key-storage-interface.md#overview)
```

Comment:
The interface is an implementation?

Reply:
Both you and @texastony agree with this.
Thinking of `Interface` as an adjective that modifies the noun `Key Storage` works for me. Rather than thinking of the noun as `Key Storage Interface.`

Changing.